staging/copy.sh

```diff
@@ -186,6 +187,7 @@ find "${CLIENT_REPO_TEMP}" -type f \( \
 
 echo "remove cyclical godep"
 rm -rf "${CLIENT_REPO_TEMP}/_vendor/k8s.io/client-go"
+rm -rf "${CLIENT_REPO_TEMP}/staging"
```

Comment:
It's certainly fine to delete that dir. But this pretty much looks like a bug of gedep, doesn't it? If we decide to live with it, we should put a comment above the "rm -rf". 

Also adding a comment, why we add staging to the GOPATH in the first place would be good.